### PR TITLE
Update manpage with new domain

### PR DESCRIPTION
**Pull request by GrayHatter**
This change is [Reviewable](https://reviewable.io/reviews/utox/utox/652)

### CODE REVIEW

File: src/utox.1 [hunk 1]
```diff
@@ -96,6 +96,6 @@ Please report bugs on
 The website for
 .B µTox
 can be found at
-.I http://utox.org/
+.I http://utox.io/
```
Comment (redmanmale) [inline src/utox.1]: https

**Comment by robinlinden** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/utox/utox/652)*

**Comment by GrayHatter** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[src/utox.1, line 99 at r1](https://reviewable.io:443/reviews/utox/utox/652#-Kb9wqGD-OG_giLGCAzC-r1-99:-KbBwO6R0iZpY7CCrFb_:b-896fix) ([raw file](https://github.com/utox/utox/blob/19d79a1fde9eeeff2fe64e787d52419f6a2f6000/src/utox.1#L99)):*
<details><summary><i>Previously, redmanmale wrote…</i></summary><blockquote>

https
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/utox/utox/652)*

**Comment by cebe** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/utox/utox/652#-:-KbBxaybDltYq11kC10j:bnfp4nl)*

**Comment by robinlinden** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/utox/utox/652#-:-KbByxLg_z7kyBE4w4Iu:bnfp4nl)*

**Pull request merged**